src/pages/patientView/mutation/MutationInformationContainer.tsx

```diff
@@ -17,7 +69,11 @@ export default class MutationInformationContainer extends React.Component<IMutat
                 visible: "excluded"
             },
             tumors: {
-                name: "Tumors"
+                name: "Tumors",
+                formatter: TumorColumnFormatter.makeRenderFunction(this.props.sampleOrder, this.props.sampleColors, this.props.sampleLabels,
```

Comment:
I think here what makes things complicated is, again, the additional data/props that need to be passed to the column formatter. For most of the columns, the basic table data is good enough for rendering purposes. But here actually we need more than a single `data:IColumnFormatterData` parameter.

So, same design question applies here: Should `TumorColumnFormatter` fetch its own additional data or not? If not, then we need to come up with a better interface design if we want to keep all column formatters simple and consistent.